src/ComposerRequireChecker/DependencySuggester/DependencySuggester.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace ComposerRequireChecker\DependencySuggester;
+
+
+use ComposerRequireChecker\DefinedSymbolsLocator\LocateDefinedSymbolsFromExtensions;
+
+class DependencySuggester
+{
+
+    public function __invoke($symbolName) : \Generator
+    {
+        yield from $this->suggestFromLoadedExtensions($symbolName);
+    }
+
+    private function suggestFromLoadedExtensions($symbolName) : \Generator
+    {
+
+        $loadedExtensions = get_loaded_extensions();
```

Comment:
I think I should completely pull this function `suggestFromLoadedExtensions` out of there, and create a standalone `FromLoadedExtensionSuggester`, as I would like to be able to add more "Suggestors" over time (e.g. as mentioned in #7 ).
But for now I'm not sure what would be the best way to accomplish this.

btw. is it "Suggester" or "Suggestor" or is it wrong anyway?! :smile: